src/lakefs_spec/spec.py

```diff
@@ -70,6 +75,24 @@ def ensure_branch(client: LakeFSClient, repository: str, branch: str, source_bra
         pass
 
 
+def get_client_blockstore_type(client: LakeFSClient) -> str:
+    """
+    Get the blockstore type from the client.
+    Backwards compatible to breaking config_api change in lakeFSClient version 0.110.1.
```

Comment:
Two things:

- The blockstore type is a server/deployment attribute, which the client is used for to query. Please change the name to `get_blockstore_type`.
- I believe the change came in `v0.111.0` (you also have it correct there down below).

Reply:
Also, maybe move this to the `util` module as well?